src/intro-to-c/arrays/arrays-intro.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Array Introduction
+author:
+    - Bed Pandey
+    - Carson Cox
+---
+
+Arrays can be used to represent a collection of items with the same datatype. For instance, let's say we wanted an array to represent the grades our class got for a test in our hypothetical class of size 5.
+
+```
++------+------+------+------+------+
+| 92.5 | 52.4 | 75.0 | 82.0 | 88.9 |
++------+------+------+------+------+
+```
+
+This is a common representation of an array, a one-dimensional grid with a specific size. Since our array should have 5 grades, we have 5 separate cells we can put our data in.
+
+But how do we do this in C?!
+
+## Creating Arrays
+To create an array, we need to specify the datatype, a variable name, and the size of the array:
```

Comment:
Before the example, I would add in a generic form as well, like "`<datatype> <name>[<size>]`," or something along those lines.